### DISCUSSION TITLE
what is the time complexity to search for a value in a v map?

### DISCUSSION THREAD

**Discussion by timi5691** [vlang/v#14978] [General]
what is the time complexity to search for a value in a v map?

**Comment by medvednikov** [comment]
O(1)

**Reply by timi5691** [reply]
thanks, I am making a game with gg library, should i just use map to store data for my game? Or should I use another way?

**Reply by JalonSolov** [reply]
That is something you will have to decide.  Try one way.  If it works for you, great.  If not, try something else.  Profiling and benchmarking can help the decision.
